asv/commands/run.py

```diff
@@ -52,6 +54,14 @@ def setup_arguments(cls, subparsers):
             "run", help="Run a benchmark suite",
             description="Run a benchmark suite.")
 
+        def positive_int(string):
+            try:
+                value = int(string)
+                if not value > 0:
+                    raise argparse.ArgumentTypeError("%r is not a positive integer" % (string,))
+            except ValueError:
+                raise argparse.ArgumentTypeError("%r is not an integer" % (string,))
+
```

Comment:
Can you put this in `asv/commands/common_args.py` so it can be used by other commands in the future?